danceschool/vouchers/views.py

```diff
@@ -63,9 +63,10 @@ def form_valid(self,form):
         logger.info('Processing gift certificate.')
 
         try:
+            currency_symbol = getConstant('general__currencySymbol')
             voucher = Voucher.create_new_code(
                 prefix='GC_',
-                name=_('Gift certificate: %s%s for %s' % (getConstant('general__currencySymbol'),self.amount, emailTo)),
+                name=_('Gift certificate: %s%s for %s' % (currency_symbol, self.amount, emailTo)),
```

Comment:
But using named placeholders is a best practice, to give context to translators and allow then to change order:
```suggestion
                name=_('Gift certificate: %(currency_sumbol)s%(amount)s for %(email_recipient)s') % (currency_symbol, self.amount, emailTo),
```

And once you’re there, you ask yourself why you’re using not-nice % format instead of the cool, modern format:
```suggestion
                name=_('Gift certificate: {currency_symbol}{amount} for {email_recipient}').format(x=y, etc),
```